src/constraints.jl

```diff
@@ -727,3 +727,313 @@ function list_of_constraint_types(model::Model)
     return Tuple{DataType, DataType}[(jump_function_type(model, f), s)
                                      for (f,s) in list]
 end
+
+
+"""
+    perturbation_range_of_feasibility(constraint::ConstraintRef)
+
+Gives the range by which the rhs coefficient can change and the current lp-basis
```

Comment:
What is the definition of the rhs coefficient of a constraint? (I know what it is, readers might not.)